platform-stack/docker-compose.yml

```diff
@@ -117,22 +113,22 @@ services:
       - local-keycloak
 
   postgres:
-    image: postgres:13.2
+    image: postgres:15.3
     container_name: postgres
     volumes:
-      - ./postgres:/docker-entrypoint-initdb.d
+      - ./postgres/:/docker-entrypoint-initdb.d/
+      - ./data:/var/lib/postgresql/data
```

Comment:
we should add the data folder to .gitignore: `/platform-stack/data/`

Reply:
done